packages/server/src/tests/banners/channelBannerTests.ts

```diff
@@ -45,7 +46,11 @@ export const BannerPaths: {
     [BannerTemplate.DigitalSubscriptionsBanner]: digiSubs.endpointPathBuilder,
     [BannerTemplate.GuardianWeeklyBanner]: guardianWeekly.endpointPathBuilder,
     [BannerTemplate.ResearchSurveyBanner]: researchSurveyBanner.endpointPathBuilder,
+<<<<<<< HEAD
+    [BannerTemplate.AuBrandMomentBanner]: auBrandMomentBanner.endpointPathBuilder,
+=======
     [BannerTemplate.SignInPromptBanner]: signInPromptBanner.endpointPathBuilder,
+>>>>>>> 55ccd1e02a599b1bd11587b886c4d4314125264a
```

Comment:
think we can accept both changes here!

Reply:
(I think this is the only thing now causing the build to fail)

Reply:
Damn! I thought I fixed all those. Weird... 😕

Reply:
Thanks 😄